docs/database/database-spans.md

```diff
@@ -268,6 +302,105 @@ Placeholders in a parameterized query SHOULD not be sanitized. E.g. `where id =
 e.g. from `IN (?, ?, ?, ?)` to `IN (?)`, as this can help with extremely long IN-clauses,
 and can help control cardinality for users who choose to (optionally) add `db.query.text` to their metric attributes.
 
+## Generating a summary of the query text
+
+The `db.query.summary` attribute captures a shortened representation of a query text
+which SHOULD have low-cardinality and SHOULD NOT contain any dynamic or sensitive data.
+
+> [!NOTE]
+> The `db.query.text` attribute is intended to identify individual queries. Even though
+> it is sanitized if captured by default, it could still have high cardinality and
+> might reach hundreds of lines.
+>
+> The `db.query.summary` is intended to provide a less granular grouping key that
+> can be used as a span name or a metric attribute in common cases. It SHOULD
+> only contain information that has a significant impact on the query, database,
+> or application performance.
```

Comment:
i'm not sure this part helps

I think I understand, that it groups things where queries in each bucket will have somewhat similar performance characteristics to each other, but worried that people would use this language to pull in more stuff into the summary than we want (increasing cardinality and reducing brevity), since there are many things that can have a significant impact (e.g. where clauses)

```suggestion
> can be used as a span name or a metric attribute in common cases.
```